src/features/auth/sign-in/screen/index.tsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import { View } from 'react-native';
+
+import Logo from '../components/Logo';
```

Comment:
Can you use an absolute import here?

Reply:
`features/auth/sign-in/components/Logo` it's going to be a little bet longer than using that